internal/reports/report_helpers.go

```diff
@@ -383,40 +507,53 @@ func fillRowStats(conv *internal.Conv, srcTable string, badWrites map[string]int
+		CategoryDescription: "No appropriate Spanner type"},
+	internal.Numeric:              {Brief: "Spanner does not support numeric. This type mapping could lose precision and is not recommended for production use", severity: warning, Category: "NUMERIC_USES"},
+	internal.NumericThatFits:      {Brief: "Spanner does not support numeric, but this type mapping preserves the numeric's specified precision", severity: suggestion, Category: "NUMERIC_THAT_FITS"},
+	internal.Decimal:              {Brief: "Spanner does not support decimal. This type mapping could lose precision and is not recommended for production use", severity: warning, Category: "DECIMAL_USES"},
+	internal.DecimalThatFits:      {Brief: "Spanner does not support decimal, but this type mapping preserves the decimal's specified precision", severity: suggestion, Category: "DECIMAL_THAT_FITS"},
+	internal.Serial:               {Brief: "Spanner does not support autoincrementing types", severity: warning, Category: "AUTOINCREMENTING_TYPE_USES"},
+	internal.AutoIncrement:        {Brief: "Spanner does not support auto_increment attribute", severity: warning, Category: "AUTO_INCREMENT_ATTRIBUTE_USES"},
+	internal.Timestamp:            {Brief: "Spanner timestamp is closer to PostgreSQL timestamptz", severity: suggestion, batch: true, Category: "TIMESTAMP_SUGGESTION"},
+	internal.Datetime:             {Brief: "Spanner timestamp is closer to MySQL timestamp", severity: warning, batch: true, Category: "TIMESTAMP_WARNING"},
+	internal.Time:                 {Brief: "Spanner does not support time/year types", severity: warning, batch: true, Category: "TIME_YEAR_TYPE_USES"},
+	internal.Widened:              {Brief: "Some columns will consume more storage in Spanner", severity: warning, batch: true, Category: "STORAGE_WARNING"},
+	internal.StringOverflow:       {Brief: "String overflow issue might occur as maximum supported length in Spanner is 2621440", severity: warning, Category: "STRING_OVERFLOW_WARNING"},
+	internal.HotspotTimestamp:     {Brief: "Timestamp Hotspot Occured", severity: warning, Category: "TIMESTAMP_HOTSPOT"},
+	internal.HotspotAutoIncrement: {Brief: "Autoincrement Hotspot Occured", severity: warning, Category: "AUTOINCREMENT_HOTSPOT"},
+	internal.InterleavedOrder: {Brief: "can be converted as Interleaved with Table", severity: suggestion, Category: "INTERLEAVE_TABLE_SUGGESTION",
+		CategoryDescription: "Some tables can be interleaved"},
+	internal.RedundantIndex:     {Brief: "Redundant Index", severity: warning, Category: "REDUNDANT_INDEX"},
+	internal.AutoIncrementIndex: {Brief: "Auto increment column in Index can create a Hotspot", severity: warning, Category: "AUTO-INCREMENT_INDEX"},
+	internal.InterleaveIndex: {Brief: "can be converted to an Interleave Index", severity: suggestion, Category: "INTERLEAVE_INDEX_SUGGESTION",
+		CategoryDescription: "Some columns can be interleaved"},
```

Comment:
Rephrase the category description: Some columns can be converted to interleave index